### ISSUE TITLE
Data set partitioning

### ISSUE THREAD

**Issue by yapingzhao** [tensorflow/nmt#337]
Hi,
I have a problem with data set partitioning: the total parallel corpus is 160,000. What should the training set, validation set and test set be? If the total parallel corpus is 1.2 million. What are the training set, validation set and test set?

Looking forward to your advice or answers.
Best regards,

yapingzhao

**Comment by luozhouyang** [comment]
A typical `train`, `dev` and `test` dataset ratios are `0.7`, `0.15`, `0.15`
